dev/app/builtin/stories/refinement-list.stories.js

```diff
@@ -30,16 +27,13 @@ export default () => {
         window.search.addWidget(
           instantsearch.widgets.refinementList({
             container,
-            attributeName: 'brand',
+            attribute: 'brand',
             operator: 'or',
             limit: 3,
-            templates: {
-              header: 'Brands with show more',
-            },
             showMore: {
               templates: {
-                active: '<button>Show less</button>',
-                inactive: '<button>Show more</button>',
+                active: 'Show less',
+                inactive: 'Show more',
```

Comment:
I thought at some point we discussed about putting all the templates together rather than having multiple templates across options.

Reply:
It would make sense. Do you mean something like this?

```js
{
  templates: {
    showMoreActive: '',
	showMoreInactive: '',
  }
}
```

Reply:
Yes - same for `searchable`. But we can do it in another pass no worries.